django_project/core/base_static/js/custom-jquery.js

```diff
@@ -35,7 +35,17 @@ $(document).ready(function () {
             }
         }
     });
-
+	
+	$('#icons-nav a').click(function () {
```

Comment:
there is something wrong with this logic
you could try to change like this

$('#icons-nav a').click(function () {
        if ($(this).hasClass("active")) {
            $(this).removeClass("active");
            $(".search-block").hide();
        }
        else {
            $(this).addClass("active");
            $(".search-block").show();
        }
    });